.github/workflows/create-release-draft.yml

```diff
@@ -13,6 +13,11 @@ on:
         description: runtime
         required: true
         default: true
+      release_enclave:
```

Comment:
Does it always contain `enclave.signed.so` and the worker binary?

Reply:
Don't get what you mean here. The package will be created by `./tee-worker/scripts/litentry/release/generate_release_pkg.sh`. So `enclave.signed.so` and worker binary are created together.

Reply:
Is there any case we only want to release one of them?

Similar to substrate client and runtime

Reply:
Yes, if you only select `enclave`, it will release only the `enclave` 
```
          elif [ "${{ github.event.inputs.release_enclave }}" = "true" ]; then
            release_type="enclave"
          elif [ "${{ github.event.inputs.release_enclave }}" = "true" ] && [ "${{ github.event.inputs.release_client }}" = "true" ] && [ "${{ github.event.inputs.release_runtime }}" = "true" ]; then
            release_type="all"
```

Reply:
It will only release `enclave` but the enclave still contains both binary (integritee-worker) and enclave (.so), right? I mean is there any case we only want to release one of them. Like only the worker part but the enclave doesn't change.

That's why I said "similar to substrate client and runtime".

Not saying we should differentiate them, I'm just raising the question

Reply:
I see, As of now, it doesn't do that. Both `integritee-worker` and `enclave` are packaged together.